gateway/xgress_edge/dialer.go

```diff
@@ -101,5 +102,5 @@ func (dialer *dialer) Dial(destination string, sessionId *identity.TokenId, addr
 
 	start := edge.NewStateConnectedMsg(result.ConnId)
 	start.ReplyTo(reply)
```

Comment:
Can we return the PeerData here to make it work the same as the xgress_edge_transport impl?

Reply:
The public key that would need to be sent back that would make up the `peerData` is stored on the `terminator` that was added when the SDK client sent a bind request. So right now the public key is part of the static terminator peerData stored in the controller.

To do what you are asking would require changing storing the public key in memory on the router instead. 

Would that also trigger the removal of the peerData storage in the controller for the terimnator?

Reply:
Will do this in a follow up.

Reply:
https://github.com/openziti/edge/issues/266